lib/dockerize_rails/docker_commands/docker_build.rb

```diff
@@ -60,13 +78,26 @@ def self.build_postgres
       end
 
       def self.build_docker_image(dockerfile, repo)
-        image = Docker::Image.build_from_dir(
-          '.',
-          dockerfile: dockerfile
-        )
+        image = Docker::Image.build_from_dir('.', dockerfile: dockerfile) do |v|
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 'v' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)

<!-- Comment by [SideCI](https://sideci.com) -->